client/question/components/LearnerViewDirective.js

```diff
@@ -1007,6 +1069,8 @@ tie.directive('learnerView', [function() {
          * question to its original state.
          */
         $scope.resetCode = function() {
+          $scope.speechBalloonsList =
```

Comment:
Ditto -- I don't believe this is used. Delete?

Reply:
See above.

Reply:
Ditto.

Reply:
Removed